### PR TITLE
Feature/radsec

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Adds built-in support for RADSEC 

# Impacts
FreeRADIUS configuration (changed auth.conf and acct.conf to use template toolkit)

# Delete branch after merge
YES

# NEWS file entries
## New Features
* Added support for RADSEC

# UPGRADE file entries

The templates `/usr/local/pf/conf/radiusd/auth.conf` and `/usr/local/pf/conf/radiusd/acct.conf` were modified to use TemplateToolkit, you will need to replace them by `/usr/local/pf/conf/radiusd/auth.conf.example` and `/usr/local/pf/conf/radiusd/acct.conf.example`, make sure to re-edit the new file and add your certificate if needed.

### CODE REVIEW

**Comment by fdurand** [comment]
Didn't find anything, i am agree to merge.

**Comment by julsemaan** [comment]
@louismunro this is waiting for your review love

File: conf/radiusd/auth.conf.example [hunk 1]
```diff
@@ -10,11 +10,44 @@ listen {
...
+              lifetime = 0
+              idle_timeout = 30
+        }
+
+        tls {
+                private_key_file = /usr/local/pf/raddb/certs/server.key
+                certificate_file = /usr/local/pf/raddb/certs/server.crt
+                ca_file = /usr/local/pf/raddb/certs/ca.pem
+                dh_file = ${certdir}/dh
+                fragment_size = 8192
```
Comment (louismunro) [inline conf/radiusd/auth.conf.example:39]: Where does this value come from? 
It seems a tad large.

File: conf/radiusd/auth.conf.example [hunk 2]
```diff
@@ -10,11 +10,44 @@ listen {
...
+        proto = tcp
+        virtual_server = packetfence
+
+        limit {
+              max_connections = 16
+              lifetime = 0
+              idle_timeout = 30
+        }
+
+        tls {
```
Comment (louismunro) [inline conf/radiusd/auth.conf.example:34]: Is there any way to have a single configuration for tls instead of here and in the eap module config?

**Review by louismunro — COMMENTED** [review]
Other than my (to be expected) rantings, it's all good.

**Pull request merged**